alchemical-setup.py

```diff
@@ -85,8 +87,19 @@ def extractSection(self, section):
          else:
 	    if O.verbose:
                print "%s\n%s\n%s" % (('Molecule '+self.ID).center(36,'='), section_string, '='*36)
-            print section_string
-            return numpy.loadtxt(StringIO(section_string), dtype=str, skiprows=1, comments=';')
+            print section_string, "section_string"
```

Comment:
The indentation here is looking odd. Either:
* You are using tabs in some places instead of spaces (which should be fixed, if so!), or:
* The print statements are not inside the `if O.verbose` section